servicebox-app/src/main/java/io/halkyon/resource/page/ServiceResource.java

```diff
@@ -0,0 +1,32 @@
+package io.halkyon.resource.page;
+
+import io.halkyon.Templates;
+import io.quarkus.qute.TemplateInstance;
+import org.jboss.resteasy.annotations.Form;
+
+import javax.transaction.Transactional;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.Response;
+
+
+@Path("/service")
```

Comment:
If we decide to group all the REST Verbs under the same endpoint `/services`, then we will refactor `/claims` and `/services` in a separate PR. We can keep this class until now.